homeless-templates/rhche-redirector.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: rhche-redirector
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
+    generation: 1
+    labels:
+      run: rhche-redirector
+    name: rhche-redirector
+  spec:
+    replicas: 1
+    selector:
+      run: rhche-redirector
+    strategy:
+      resources:
+        requests:
+          cpu: 20m
+          memory: 20Mi
+        limits:
+          cpu: 200m
+          memory: 200Mi
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          run: rhche-redirector
+      spec:
+        containers:
+        - env:
+          - name: REDIRECTOR_DESTINATION
+            valueFrom:
+                configMapKeyRef:
+                  name: rhche-redirector
+                  key: redirector.destination
+          - name: REDIRECTOR_TYPE
+            valueFrom:
+                configMapKeyRef:
+                  name: rhche-redirector
+                  key: redirector.type
+          image: prod.registry.devshift.net/osio-prod/base/nginx-redirector:${IMAGE_TAG}
```

Comment:
since we are moving from dev shift maybe we should use image from quay? from what I see we have it already on quay https://quay.io/repository/openshiftio/rhel-base-nginx-redirector

Reply:
I've updated the image to use the quay registry. I've also added a staging & production image /w appropriate replica count